src/test/java/SetTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    public void testSetSize() {
+        var result = numbers.size();
+        assertThat(result).isEqualTo(3);
```

Comment:
```suggestion
assertThat(result).hasSize(3);
```
참고로, assertj에서는 Collection에 대한 검증 기능도 지원하고 있습니다! :)